src/game/Pet.cpp

```diff
@@ -143,15 +143,6 @@ bool Pet::LoadPetFromDB(Player* owner, uint32 petentry, uint32 petnumber, bool c
     uint32 summon_spell_id = fields[21].GetUInt32();
     SpellEntry const* spellInfo = sSpellStore.LookupEntry(summon_spell_id);
 
-    bool is_temporary_summoned = spellInfo && GetSpellDuration(spellInfo) > 0;
```

Comment:
there is no quest pet that are here to help you for certain time?
If player disconect and get back to it the timer should continue?